### PR TITLE
Unlist content

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/how-tos/collect-sensor-data.md [hunk 1]
```diff
@@ -130,11 +130,11 @@ Click the **Save** button in the top right corner of the page to save your confi
 
 ## Next steps
 
-Now that you have collected sensor data, you can [query it](/how-tos/sensor-data-query-with-third-party-tools/), [access it programmatically](/how-tos/sensor-data-query-sdk/) or [visualize it](/how-tos/sensor-data-visualize/) with third-party tools.
+Now that you have collected sensor data, you can [query it](/how-tos/sensor-data-query-with-third-party-tools/), [access it programmatically](/appendix/apis/components/sensor/) or [visualize it](/how-tos/sensor-data-visualize/) with third-party tools.
```
Comment (JessamyT) [inline docs/how-tos/collect-sensor-data.md:133]: Wouldn’t we want them to use the data query API, not the sensor component API?

File: docs/installation/prepare/_index.md
Comment (JessamyT) [inline docs/installation/prepare/_index.md:1]: Why this one?

File: docs/tutorials/get-started/blink-an-led.md
Comment (JessamyT) [inline docs/tutorials/get-started/blink-an-led.md:1]: Why this one?

**Review by JessamyT — COMMENTED** [review]
I think the sensor query links all need fixing to go to the data client query tabular/binary content instead

File: docs/how-tos/create-module.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ level: "Intermediate"
 date: "2024-07-30"
 # updated: ""  # When the tutorial was last entirely checked
 cost: "0"
+toc_hide: true
```
Comment (npentrel) [inline docs/how-tos/create-module.md:19]:
```suggestion
```

File: docs/installation/prepare/_index.md
Comment (npentrel) [reply docs/installation/prepare/_index.md:1]: It's accessible through the installation page and the prepare nav was always weird

File: docs/tutorials/get-started/blink-an-led.md
Comment (npentrel) [reply docs/tutorials/get-started/blink-an-led.md:1]: Hasn't been reviewed in ages and isn't really in line with our positioning anymore

File: docs/how-tos/collect-sensor-data.md (hunk 1 above)
Comment (npentrel) [inline docs/how-tos/collect-sensor-data.md:133]:
```suggestion
Now that you have collected sensor data, you can [query it](/how-tos/sensor-data-query-with-third-party-tools/), [access it programmatically](/appendix/apis/data-client/) or [visualize it](/how-tos/sensor-data-visualize/) with third-party tools.
```

File: docs/how-tos/collect-sensor-data.md [hunk 3]
```diff
@@ -130,11 +130,11 @@ Click the **Save** button in the top right corner of the page to save your confi
...
 
 {{< cards >}}
 {{% card link="/how-tos/sensor-data-query-with-third-party-tools/" %}}
-{{% card link="/how-tos/sensor-data-query-sdk/" %}}
+{{% card link="/appendix/apis/components/sensor/" %}}
```
Comment (npentrel) [inline docs/how-tos/collect-sensor-data.md:137]:
```suggestion
{{% card link="/appendix/apis/data-client/" %}}
```

File: docs/how-tos/sensor-data-query-with-third-party-tools.md [hunk 4]
```diff
@@ -281,10 +281,10 @@ db.readings.aggregate(
 
 For information on connecting to your Atlas instance from other MQL clients, see the MongoDB Atlas [Connect to your Cluster Tutorial](https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/).
 
-On top of querying sensor data with third-party tools, you can also [query it with the Python SDK](/how-tos/sensor-data-query-sdk/) or [visualize it](/how-tos/sensor-data-visualize/).
+On top of querying sensor data with third-party tools, you can also [query it with the Python SDK](/appendix/apis/components/sensor/) or [visualize it](/how-tos/sensor-data-visualize/).
```
Comment (npentrel) [inline docs/how-tos/sensor-data-query-with-third-party-tools.md:284]:
```suggestion
On top of querying sensor data with third-party tools, you can also [query it with the Python SDK](/appendix/apis/data-client/) or [visualize it](/how-tos/sensor-data-visualize/).
```

File: docs/how-tos/sensor-data-query-with-third-party-tools.md [hunk 5]
```diff
@@ -281,10 +281,10 @@ db.readings.aggregate(
...
 
 {{< cards >}}
-{{% card link="/how-tos/sensor-data-query-sdk/" %}}
+{{% card link="/appendix/apis/components/sensor/" %}}
```
Comment (npentrel) [inline docs/how-tos/sensor-data-query-with-third-party-tools.md:287]:
```suggestion
{{% card link="/appendix/apis/data-client/" %}}
```

File: docs/how-tos/sensor-data-visualize.md [hunk 6]
```diff
@@ -230,10 +230,10 @@ See the [guide on querying sensor data](/how-tos/sensor-data-query-with-third-pa
 
 For more detailed instructions on using Grafana, including a full step-by-step configuration walkthrough, see [visualizing data with Grafana](/tutorials/services/visualize-data-grafana/).
 
-On top of visualizing sensor data with third-party tools, you can also [query it with the Python SDK](/how-tos/sensor-data-query-sdk/) or [query it with the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
+On top of visualizing sensor data with third-party tools, you can also [query it with the Python SDK](/appendix/apis/components/sensor/) or [query it with the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
```
Comment (npentrel) [inline docs/how-tos/sensor-data-visualize.md:233]:
```suggestion
On top of visualizing sensor data with third-party tools, you can also [query it with the Python SDK](/appendix/apis/data-client/) or [query it with the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
```

File: docs/how-tos/sensor-data-visualize.md [hunk 7]
```diff
@@ -230,10 +230,10 @@ See the [guide on querying sensor data](/how-tos/sensor-data-query-with-third-pa
...
 
 {{< cards >}}
-{{% card link="/how-tos/sensor-data-query-sdk/" %}}
+{{% card link="/appendix/apis/components/sensor/" %}}
```
Comment (npentrel) [inline docs/how-tos/sensor-data-visualize.md:236]:
```suggestion
{{% card link="/appendix/apis/data-client/" %}}
```

File: docs/services/data/_index.md [hunk 8]
```diff
@@ -93,11 +93,11 @@ Upload data to the Viam cloud from your computer or mobile device using the data
 ### Query your data
 
 Once your data has [synced](/services/data/capture-sync/), you can query it using the [data client API](/appendix/apis/data-client/).
-For _tabular_ sensor data, you can also [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
+For _tabular_ sensor data, you can also [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/appendix/apis/components/sensor/) from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
```
Comment (npentrel) [inline docs/services/data/_index.md:96]:
```suggestion
For _tabular_ sensor data, you can also run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
```

File: docs/services/data/_index.md [hunk 9]
```diff
@@ -93,11 +93,11 @@ Upload data to the Viam cloud from your computer or mobile device using the data
...
 
 {{< cards >}}
 {{% card link="/how-tos/sensor-data-query-with-third-party-tools/" noimage="True" %}}
-{{% card link="/how-tos/sensor-data-query-sdk/" noimage="True" %}}
+{{% card link="/appendix/apis/components/sensor/" noimage="True" %}}
```
Comment (npentrel) [inline docs/services/data/_index.md:100]:
```suggestion
{{% card link="/appendix/apis/data-client/" noimage="True" %}}
```

**Review by JessamyT — APPROVED** [review]
Lgtm

**Pull request merged**